### PR TITLE
Code refactoring part 1

### LINKED ISSUE #410: Convert filenames from CamelCase to snake_case (gui directory)

### LINKED ISSUE #411: Convert filenames from CamelCase to snake_case (bricks directory)
Maybe remove prefix Bricks.
For example:
* ApertureBrick.py -> aperture.py
* BeamstopDistanceBrick.py -> beamstop_distance.py

### PR DESCRIPTION
**Pull request by IvarsKarpics**
Dear all,

This is the first PR to start mxcube qt code refactoring and cleans up the initialization part of qt app.

### CODE REVIEW

**Comment by rhfogh** [comment]
@IvarsKarpics 
An issue to discuss:
I notice you have renamed all the files in gui/utils to snake_case. This is a major change - until now files with snake_case names were a few, rare, exceptions (like queue_model_objects). So:
- When was this change discussed - or agreed?
- What is its scope - do we intend all files to be renamed? or if not which ones?

I know it says in PEP-8 that package names should be "short, all-lower-case names, and the use of underscores is discouraged". But 1) Your new versions have underscores, 2) The things we need to name are not susceptible to short single-word names (what is the alternative to queue_model_objects?) 3) at a guess 95%+ of all packages have CamelCase names at present.
I can live with any convention, and I am willing to rename all the files and all the imports in the name of consistency, if it comes to that. But we ought to be sure that we all agree where we are going before we get on that train.

Maybe we want some input also from others, including the ESRF people?

File: configuration/example_mxcube_gui.yml [hunk 1]
```diff
@@ -670,8 +686,8 @@
                       - {comment: Allow to control brick in slave mode, default_value: 0,
                         hidden: false, name: instanceAllowConnected, old_value: 0,
                         type: boolean, value: 0}
-                      - {comment: '', default_value: /sample-view, hidden: false, name: mnemonic,
-                        old_value: /sample-view, type: string, value: /sample-view}
+                      - {comment: '', default_value: /graphics, hidden: false, name: mnemonic,
```
Comment (rhfogh) [inline configuration/example_mxcube_gui.yml:689]: In some context we are changing from graphics to sample_view. Is this place an exception?

**Comment by IvarsKarpics** [comment]
Hi @rhfogh 

I mentioned few times during the last developers meeting that also mxcube qt needs a refactoring. Currently there issues in the repository: 
- Mix of camel case and snake case module names.
- Some class names are the same like module names.
- Re declared or unused variables.
PR should not be accepted immediately, I can change it to WIP and we can discuss this during the next developers meeting. Or we have a separate developers meeting for those who are interested in mxcube qt.

**Comment by rhfogh** [comment]
If we want consistency - which would indeed be a very good thing - I think it would be better if we had a wide agreement on what measures to take, and across what folders. Moving from < 5% of packages being snake_case to 35% of packages being snake_case is not necessarily a win for consistency, and I think it might be hard to get towards 100% (and rename all the files)  unless it is clear that it is a commoon decision. If this has already been decded and I just missed it, I apologis.

Maybe raise it as an issue, rather than wait for the next meeting?

**Review by jordiandreu — APPROVED** [review]
In agreement with what I have already pointed out in #410 I'm fine with these changes.

**Pull request merged**